File: 프로그래머스/0/120923. 연속된 수의 합/연속된 수의 합.java
import java.util.*;
class Solution {
    public List <Integer> solution(int num, int total) {
        List <Integer> answer = new ArrayList();
        
        int minus = num/2;
        int check = 0;
        if(total % num ==0)check = total / num;
        else check = total / num+1;
        int start = check-minus;
        
        for(int i = start ; i < start+num ; i++){
            answer.add(i);
        }
        
        return answer;
    }
}

File: C#/프로그래머스/0/120923. 연속된 수의 합/연속된 수의 합.cs
using System;

public class Solution {
    public int[] solution(int num, int total) {
            int[] answer = new int[num];

            int start = 0;
            if (num % 2 == 1) start = total / num - num / 2;
            else start = (total / num + 1) - num / 2;

            for (int i = 0; i < num; i++)
                answer[i] = start + i;
            return answer;
    }
}